### ISSUE TITLE
Loss graph lost when training continues on colab

### ISSUE THREAD

**Issue by vanquang203546** [AlexeyAB/darknet#8840]
If you want to report a bug - provide:
    * description of a bug
    * what command do you use?
    * do you use Win/Linux/Mac?
    * attach screenshot of a bug with previous messages in terminal
    * in what cases a bug occurs, and in which not?
    * if possible, specify date/commit of Darknet that works without this bug
    * show such screenshot with info
```
./darknet detector test cfg/coco.data cfg/yolov4.cfg yolov4.weights data/dog.jpg
 CUDA-version: 10000 (10000), cuDNN: 7.4.2, CUDNN_HALF=1, GPU count: 1
 CUDNN_HALF=1
 OpenCV version: 4.2.0
 0 : compute_capability = 750, cudnn_half = 1, GPU: GeForce RTX 2070
net.optimized_memory = 0
mini_batch = 1, batch = 8, time_steps = 1, train = 0
   layer   filters  size/strd(dil)      input                output
```

**Comment by stephanecharette** [comment]
This is fixed on the new darknet/yolo repo:  https://github.com/hank-ai/darknet

**Comment by Dickoabc123** [comment]
I am getting the problem similar to that, where I am only getting the mAP for every 1000 iterations, surely there must be a way to specify how many iterations to show the mAP. Would be better for every Epoch.  Does anyone know how to show the mAP sooner than every 1000 epochs?

**Comment by stephanecharette** [comment]
1) Darknet doesn't work with epochs.  2) You should use the Hank.ai repo instead.  https://github.com/hank-ai/darknet#table-of-contents

**Comment by Dickoabc123** [comment]
@stephanecharette thank you for that. Does that give the validation losses ?
Since I already have my darker files set up and configured, is it easy enough to transfer across to Hank.ai ?
